### PR TITLE
Hrajchert/scp 4727 remove playground common

### PR DESCRIPTION
**Pull request by hrajchert**
This PR removes the dependency on playground common and unlocks [PR-18](https://github.com/input-output-hk/marlowe-playground/pull/18)

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
    - [ ] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] Reviewer requested

### CODE REVIEW

File: marlowe-playground-server/app/FlowChartGenerator.hs [hunk 1]
```diff
@@ -17,16 +17,16 @@ import qualified ZeroCouponBond as ZeroCouponBond
 main :: IO ()
 main = do
     putStrLn "===== SIMPLE ESCROW ====="
-    putStrLn $ toMermaid Escrow.contract
+    putStrLn $ toMermaid Escrow.escrow
```
Comment (hrajchert) [inline marlowe-playground-server/app/FlowChartGenerator.hs:20]: I had to change `contract` => for the name of the contract as the Marlowe extended package is now exporting a contract identifier

File: marlowe-playground-server/src/System/IO/Extras.hs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+-- withFile is a clone of System.IO.withFile however it is generalized over the monad
+-- This might be done using unliftio I think however it looked to be more pain that
+-- it was worth so I simply copied and pasted the definitions and changed the types.
+withFile
+    :: (MonadMask m, MonadIO m)
+    => FilePath
+    -> IOMode
+    -> (Handle -> m a)
+    -> m a
+withFile path mode =
```
Comment (jhbertra) [inline marlowe-playground-server/src/System/IO/Extras.hs:16]: Might want to look in `lifted-base` to see if this is defined there

File: marlowe-playground-server/app/PSGenerator.hs [hunk 3]
```diff
@@ -152,12 +143,14 @@ myBridge :: BridgePart
...
-    PSGenerator.Common.servantBridge <|>
-    PSGenerator.Common.miscBridge <|>
+    -- FIXME: Just commenting these, not removing until I make SCP-4726 to see that it
+    --        does not affect the entire build
+    -- PSGenerator.Common.aesonBridge <|>
+    -- PSGenerator.Common.containersBridge <|>
+    -- PSGenerator.Common.languageBridge <|>
+    -- PSGenerator.Common.ledgerBridge <|>
+    -- PSGenerator.Common.servantBridge <|>
+    -- PSGenerator.Common.miscBridge <|>
```
Comment (jhbertra) [inline marlowe-playground-server/app/PSGenerator.hs:153]: Why comment out instead of deleting? Is this to be replaced with something else down the road?

**Review by jhbertra — APPROVED** [review]
LGTM (when it builds)

**Pull request merged**